docs/sources/setup-grafana/configure-security/configure-authentication/azuread/index.md

```diff
@@ -21,6 +21,10 @@ weight: 800
 
 The Azure AD authentication allows you to use an Azure Active Directory tenant as an identity provider for Grafana. You can use Azure AD application roles to assign users and groups to Grafana roles from the Azure Portal.
 
+{{% admonition type="note" %}}
+If Users use the same email address in Azure AD that they use with other authentication providers (such as Grafana.com), you need to do additional configuration to ensure that the users are matched correctly. Please refer to [Using the same email address to login with different identity providers]({{< relref "../../configure-authentication#using-the-same-email-address-to-login-with-different-identity-providers" >}}) for more information.
+{{% /admonition %}}
```

Comment:
this admonition that was fixed in the original PR is brand new to this branch. Is that expected?

Reply:
This applies to 10.4

Reply:
yes, this belongs in 10.4, confirmed with Misi